zone/matches.php

```diff
@@ -966,7 +973,9 @@ public function draw(int $user_id): array
             $match_ids = [];
             $user_ids = [];
             $factor = self::get_draw_factor();
-            $draw_matches = zone_util::draw_teams()->make_matches($players_list, $factor);
+            $switch_player_numbers = self::get_draw_switch_player_numbers();
+            $draw_matches = zone_util::draw_teams()->make_matches($players_list, $factor,
```

Comment:
wuerde schoener finden wenn alle argumente umbrechen oder gar keins ^^ in dem fall alle, sonst ist die zeile so lang

Reply:
:heavy_check_mark: